examples/Custom_Hooks/campaign_stopping.py

```diff
@@ -0,0 +1,238 @@
+## Campaign Stopping
+
+# This example demonstrates how to use custom hook mechanics to stop unpromising
```

Comment:
I'd also include that one can use this to stop "sufficiently good" campaigns and phrase this more positively.

Reply:
added